test/test-main.js

```diff
@@ -0,0 +1,12 @@
+var main = require("./main");
+
+exports["test main"] = function(assert) {
+  assert.pass("Unit test running!");
```

Comment:
This file was built automatically on creating the add-on, I too think it's unnecessary to have a different testing framework. However, I'll check the docs to see if there are any advantages of this.